components/compliance-service/scanner/db.go

```diff
@@ -104,45 +105,44 @@ func isStatusUpdateValid(oldStatus, newStatus string) bool {
 	return true
 }
 
-func (s *Scanner) UpdateJobStatus(job_id string, newStatus string, startTime *time.Time, endTime *time.Time) {
+func (s *Scanner) UpdateJobStatus(job_id string, newStatus string, startTime *time.Time, endTime *time.Time) error {
 	// get current status of job
 	var status string
 	err := s.DB.QueryRow(sqlGetNodeStatus, job_id).Scan(&status)
 	if err != nil {
-		logrus.Errorf("UpdateJobStatus unable to get current job status: %s", err.Error())
-		return
+		return errors.Wrapf(err, "UpdateJobStatus unable to get current job status")
```

Comment:
i still need to follow up with a commit that ensures all the callers of these functions are at the very least logging the errors found

Reply:
that's done now, many thanks to the linter!

Reply:
👍 We weren't brave enough to fix this error handling when moving the code around, thanks for taking care of it.